dask/dataframe/core.py

```diff
@@ -7965,7 +7967,12 @@ def idxmaxmin_row(x, fn=None, skipna=True):
         value = [getattr(x.value, minmax)(skipna=skipna)]
     else:
         idx = value = meta_series_constructor(x)([], dtype="i8")
-    return meta_frame_constructor(x)({"idx": idx, "value": value})
+    return meta_frame_constructor(x)(
+        {
+            "idx": pd.Series(idx, dtype=x.index.dtype),
+            "value": pd.Series(value, dtype=x.dtypes.iloc[0]),
```

Comment:
I think helpers like `meta_frame_constructor` / `meta_series_constructor` are being used here for better GPU support (xref https://github.com/dask/dask/pull/9570). Should probably use `meta_series_constructor` here instead. 

cc @rjzamora @charlesbluca @quasiben it'd be good to have coverage in gpuCI for this. I just happened to remember https://github.com/dask/dask/pull/9570 -- otherwise it'd be easy to reintroduce `pandas`-specific constructors.

Reply:
Adjusted this pr